sky/clouds/lambda_labs.py

```diff
@@ -233,3 +235,12 @@ def accelerator_in_region_or_zone(self,
                                       zone: Optional[str] = None) -> bool:
         return service_catalog.accelerator_in_region_or_zone(
             accelerator, acc_count, region, zone, 'lambda')
+
+    @classmethod
+    def support(cls, requested_features: List[str]) -> bool:
+        if 'autostop' in requested_features:
+            print(f'{colorama.Fore.YELLOW}'
```

Comment:
Do we need this print here? Seems we can print it when we need it in the backend.